App.js

```diff
@@ -68,6 +68,20 @@ export default function App() {
         tabBarOptions={{
           activeTintColor: '#3DD5F4',
         }}>
+        <Tab.Screen
```

Comment:
nit: we should create a ticket to refactor this into navigation component in the future